src/Akeneo/Platform/Bundle/UIBundle/Resources/public/templates/menu/user-navigation.html

```diff
@@ -1,3 +1,5 @@
+<div id="dropzone-before-user-navigation"></div>
```

Comment:
We shoudl add a comment to explain the container is used in the Free Trial?
```suggestion
<div id="dropzone-before-user-navigation"><% /** Container used in Free Trial **/ %></div>
```

Or override the template directly in FT?

Reply:
As we already override the `user-navigation` module in the FT we can also override the template to add this dedicated container directly in the FT

Reply:
I don't know what is the best solution. I let you choose

Reply:
Could you move move this part in the FT, please?